src/main/java/com/couchbase/connect/kafka/CouchbaseSourceTask.java

```diff
@@ -124,19 +121,51 @@ public void start(Map<String, String> properties) {
         couchbaseReader.start();
     }
 
+    private Converter createConverter(final String className) {
+        try {
+            return (Converter) Class.forName(className).newInstance();
+        } catch (InstantiationException e) {
+            throw new ConnectException("Couldn't create message converter", e);
+        } catch (IllegalAccessException e) {
+            throw new ConnectException("Couldn't create message converter", e);
+        } catch (ClassNotFoundException e) {
+            throw new ConnectException("Couldn't create message converter", e);
+        }
+    }
+
+    private Filter createFilter(final String className) {
+        if (className != null && !"".equals(className)) {
+            try {
+                return (Filter) Class.forName(className).newInstance();
+            } catch (InstantiationException e) {
+                throw new ConnectException("Couldn't create filter in CouchbaseSourceTask due to an error", e);
+            } catch (IllegalAccessException e) {
+                throw new ConnectException("Couldn't create filter in CouchbaseSourceTask due to an error", e);
+            } catch (ClassNotFoundException e) {
+                throw new ConnectException("Couldn't create filter in CouchbaseSourceTask due to an error", e);
+            }
```

Comment:
could you extract this method as utility or try to use `org.apache.kafka.common.utils.Utils.newInstance()`